packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -121,8 +170,80 @@ class ExpansionPanelList extends StatelessWidget {
   /// The duration of the expansion animation.
   final Duration animationDuration;
 
+  // Whether multiple panels can be open simultaneously
+  final bool _allowOnlyOnePanelOpen;
+
+  @override
+  State<StatefulWidget> createState() => new _ExpansionPanelListState();
+}
+
+class _ExpansionPanelListState extends State<ExpansionPanelList> {
```

Comment:
An assertion you are missing is whether there are duplicate values in the list of expansion panels